packages/react-core/src/evm/hooks/async/marketplace.ts

```diff
@@ -1380,6 +1381,71 @@ export function useBuyNow(contract: RequiredParam<Marketplace>) {
   );
 }
 
+/**
+ * Buy a direct listing from a MarketplaceV3 contract
```

Comment:
```suggestion
 * Buy a direct listing
```
simple is better since it'll show on the build tab

Reply:
Done